stdlib/public/core/ImplicitlyUnwrappedOptional.swift

```diff
@@ -49,3 +49,73 @@ public enum ImplicitlyUnwrappedOptional<Wrapped> : ExpressibleByNilLiteral {
     self = .none
   }
 }
+
+extension ImplicitlyUnwrappedOptional : CustomStringConvertible {
+  /// A textual representation of the value, or `nil`.
+  @_inlineable // FIXME(sil-serialize-all)
+  public var description: String {
+    switch self {
+    case .some(let value):
+      return String(describing: value)
+    case .none:
+      return "nil"
+    }
+  }
+}
+
+/// Directly conform to CustomDebugStringConvertible to support
+/// optional printing. Implementation of that feature relies on
+/// _isOptional thus cannot distinguish ImplicitlyUnwrappedOptional
+/// from Optional. When conditional conformance is available, this
+/// outright conformance can be removed.
+extension ImplicitlyUnwrappedOptional : CustomDebugStringConvertible {
```

Comment:
Should we leave this one "removed"? It's not needed to fix the regression at hand.